src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -71,6 +73,46 @@ public void adsLeaderboardAndInContentOnPage(String wikiName,
   @Test(
       groups = "AdsSlotsMercury",
       dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "leaderboardSlotOnPageWithInfobox"
+  )
+  public void adsLeaderboardOnPageWithInfobox(String wikiName,
+                                               String article,
+                                               String adUnit) {
+
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
+
+    ads.verifyGptIframe(adUnit, MOBILE_TOP_LEADERBOARD, SRC);
+
+    Point leaderboardLocation = driver.findElement(By.id(MOBILE_TOP_LEADERBOARD)).getLocation();
+    Point infoboxLocation = driver.findElement(By.className("portable-infobox")).getLocation();
+
+    Assertion.assertTrue(leaderboardLocation.getY() > infoboxLocation.getY());
+  }
+
+  @Test(
+      groups = "AdsSlotsMercury",
+      dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "leaderboardSlotOnPageWithoutInfobox"
+  )
+  public void adsLeaderboardOnPageWithoutInfobox(String wikiName,
+                                              String article,
+                                              String adUnit) {
+
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
+
+    ads.verifyGptIframe(adUnit, MOBILE_TOP_LEADERBOARD, SRC);
+
+    Point leaderboardLocation = driver.findElement(By.id(MOBILE_TOP_LEADERBOARD)).getLocation();
+    Point pageHeader = driver.findElement(By.className("wiki-page-header")).getLocation();
+
+    Assertion.assertTrue(leaderboardLocation.getY() > pageHeader.getY());
```

Comment:
Per offline conversation, it makes sense to compare TOP position of leaderboard with BOTTOM position of the header